### PR TITLE
Update php packages

### PR DESCRIPTION
**Pull request by uberhacker**
Tested on all architectures.

### CODE REVIEW

File: packages/php80.rb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    #system "sed -i '98295,98296d' configure"
+    # Fix /usr/bin/file: No such file or directory
+    system 'filefix'
+  end
+
+  def self.build
+    ENV['TMPDIR'] = "#{CREW_PREFIX}/tmp"
+    ENV['CFLAGS'] = ' -liconv'
+    system './configure',
+           "--prefix=#{CREW_PREFIX}",
```
Comment (supechicken) [inline packages/php80.rb:71]: Does `CREW_OPTIONS` works?

File: packages/php80.rb [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    system 'make'
+  end
+
+  def self.check
+    #system 'make', 'test'
+  end
+
+  def self.install
+    system "mkdir -p #{CREW_DEST_PREFIX}/log"
+    system "mkdir -p #{CREW_DEST_PREFIX}/tmp/run"
```
Comment (supechicken) [inline packages/php80.rb:133]: Rubyize with `FileUtils`?

File: packages/php80.rb (hunk 1 above)
Comment (uberhacker) [reply packages/php80.rb:71]: Yes, but it requires a lot of refactoring.  I was hoping you didn't notice. :)

File: packages/php80.rb (hunk 2 above)
Comment (uberhacker) [reply packages/php80.rb:133]: Good call.

**Pull request closed without merging**